### PR TITLE
updates from training box: validate on last 30.  adjust freq of updat…

### PR DESCRIPTION
**Pull request by amj**
…e_ratio_hook.  Adjust games/generation constant.  Add notes.  Default shuffle buffer size is 20k

### CODE REVIEW

File: dual_net.py [hunk 1]
```diff
@@ -298,17 +297,18 @@ def input_fn():
     if steps is None:
         steps = EXAMPLES_PER_GENERATION // FLAGS.train_batch_size
     estimator.train(input_fn, hooks=[
-                        update_ratio_hook, step_counter_hook], steps=steps)
+        update_ratio_hook, step_counter_hook], steps=steps)
 
 
 def validate(working_dir, tf_records, checkpoint_name=None, validate_name=None):
     estimator = get_estimator(working_dir)
+    validate_name = validate_name or "selfplay"
```
Comment (sethtroisi) [inline dual_net.py]: it's weird that this has different style than the next line.

can you unify them to
A = A or default
or
if B is none: B = default

File: notes.txt [hunk 2]
```diff
@@ -24,3 +24,20 @@ buffer down to 100k
 
 1. v360ish -- entered experimental mode; freely adjusted learning rate up and
 down, adjusted batch size, etc, to see if valnet divergence could be fixed.
+
+
+== v7
+
+Started training with model 174.  Moved minimum games before starting training
```
Comment (sethtroisi) [inline notes.txt]: Rolled back to v5 model 173 and continue from there with first v7 model being 174

File: notes.txt [hunk 3]
```diff
@@ -24,3 +24,20 @@ buffer down to 100k
...
+from 5k up to 10k.
+
+1. for v215, adjusted validation set to be last 30 models instead of last 50.
```
Comment (sethtroisi) [inline notes.txt]: sudden use of v to refer to model

File: notes.txt [hunk 4]
```diff
@@ -24,3 +24,20 @@ buffer down to 100k
 
 1. v360ish -- entered experimental mode; freely adjusted learning rate up and
 down, adjusted batch size, etc, to see if valnet divergence could be fixed.
+
+
+== v7
+
+Started training with model 174.  Moved minimum games before starting training
+from 5k up to 10k.
```
Comment (sethtroisi) [inline notes.txt]: seems like it was 500

File: rl_loop.py [hunk 5]
```diff
@@ -29,10 +29,10 @@
 import shipname
 
 # How many games before the selfplay workers will stop trying to play more.
-MAX_GAMES_PER_GENERATION = 10000
+MAX_GAMES_PER_GENERATION = 20000
```
Comment (sethtroisi) [inline rl_loop.py]: we have several models with more than this so we may have self-play pauses if you choose this number

File: rl_loop.py [hunk 6]
```diff
@@ -122,7 +122,7 @@ def validate(working_dir, model_num=None, validate_name=None):
     # Run on the most recent 50 generations,
```
Comment (sethtroisi) [inline rl_loop.py]: s/50/30

File: notes.txt (hunk 2 above)
Comment (amj) [reply notes.txt]: Done

File: dual_net.py (hunk 1 above)
Comment (amj) [reply dual_net.py]: Done.

File: notes.txt (hunk 4 above)
Comment (amj) [reply notes.txt]: It wasn't :P

File: rl_loop.py (hunk 5 above)
Comment (amj) [reply rl_loop.py]: Ack.  We're not using py for selfplay anymore so /shruggie

File: rl_loop.py (hunk 6 above)
Comment (amj) [reply rl_loop.py]: Done.

File: notes.txt (hunk 3 above)
Comment (amj) [reply notes.txt]: Done.

**Pull request merged**